qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -0,0 +1,69 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+# This is the only folder in qiita_pet that should import outside qiita_pet
+# The idea is that this proxies the call and response dicts we expect from the
+# Qiita API once we build it. This will be removed and replaced with API calls
+# when the API is complete.
+from qiita_db.study import Study
+from qiita_pet.handlers.api_proxy.util import check_access
+
+
+def data_types_get_req():
+    """Returns data types available in the system
+
+    Returns
+    -------
+    list of str
+        Data types available on the system
+    """
+    data_types = Study.all_data_types()
+    return data_types
+
+
+def study_get_req(study_id, user_id):
+    """Returns information available for the given study
+
+    Parameters
+    ----------
+    study_id : int
+        Study id to get prep template info for
+    user_id : str
+        User requesting the info
+
+    Returns
+    -------
+    dict of info
```

Comment:
it's not always a dict of strings. Some are list of strings as the value, others are list of ints.